hedera-node/hedera-app-spi/src/test/java/com/hedera/node/app/spi/config/validation/BadDefinedRecord.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.config.validation;
+
+import com.hedera.node.app.spi.config.types.KeyValuePair;
+import com.swirlds.config.api.ConfigData;
+import com.swirlds.config.api.ConfigProperty;
+import java.util.List;
+
+@ConfigData
+public record BadDefinedRecord(
+        @EmulatesMap @ConfigProperty KeyValuePair pair, @EmulatesMap @ConfigProperty List<String> data) {}
```

Comment:
I don't get what it means to have `@EmulatesMap` twice here - once on each record component?

Reply:
yes and it used to test wrong usage of it

Reply:
oh, didn't catch that.